region/brinstar/red/Red Tower.json

```diff
@@ -519,24 +543,236 @@
+        "Freeze a Beetom at head height where Samus can Spinjump into it and begin Moondancing.",
+        "If needed, the Beetom can be left at the door while Samus moves to the farm bugs."
+      ]
+    },
+    {
+      "link": [1, 3],
+      "name": "Leave with Extended Moondance",
+      "notable": false,
+      "requires": [
+        {"obstaclesNotCleared": ["A"]},
+        "h_canUsePowerBombs",
+        "canExtendedMoondance",
+        "canTrickyUseFrozenEnemies",
+        {"enemyDamage": {
+          "enemy": "Beetom",
+          "type": "contact",
+          "hits": 2
+        }},
+        "h_ExtendedMoondanceBeetomLeniency"
+      ],
+      "exitCondition": {
+        "leaveWithStoredFallSpeed": {
+          "fallSpeedInTiles": 2
+        }
+      },
+      "note": [
+        "Break the Power Bomb Blocks without killing the Beetom",
+        "Freeze a Beetom at head height where Samus can Spinjump into it and begin Moondancing.",
+        "After 175 Moonfalls, reposition the Beetom to chest height.",
+        "If needed, the Beetom can be left at the door while Samus moves to the farm bugs."
+      ]
+    },
+    {
+      "link": [1, 4],
+      "name": "Leave with Moondance",
+      "notable": false,
+      "requires": [
+        {"obstaclesNotCleared": ["A"]},
+        "h_canUsePowerBombs",
+        "canMoondance",
+        "canTrickyUseFrozenEnemies",
+        {"enemyDamage": {
+          "enemy": "Beetom",
+          "type": "contact",
+          "hits": 6
+        }}
+      ],
+      "exitCondition": {
+        "leaveWithStoredFallSpeed": {
+          "fallSpeedInTiles": 1
+        }
+      },
+      "note": [
+        "Break the Power Bomb Blocks without killing the Beetom",
+        "Freeze a Beetom at head height where Samus can Spinjump into it and begin Moondancing.",
+        "If needed, the Beetom can be left at the door while Samus moves to the farm bugs."
+      ]
+    },
+    {
+      "link": [1, 4],
+      "name": "Leave with Extended Moondance",
+      "notable": false,
+      "requires": [
+        {"obstaclesNotCleared": ["A"]},
+        "h_canUsePowerBombs",
+        "canExtendedMoondance",
+        "canTrickyUseFrozenEnemies",
+        {"enemyDamage": {
+          "enemy": "Beetom",
+          "type": "contact",
+          "hits": 6
+        }},
+        "h_ExtendedMoondanceBeetomLeniency"
+      ],
+      "exitCondition": {
+        "leaveWithStoredFallSpeed": {
+          "fallSpeedInTiles": 2
+        }
+      },
+      "note": [
+        "Break the Power Bomb Blocks without killing the Beetom",
+        "Freeze a Beetom at head height where Samus can Spinjump into it and begin Moondancing.",
+        "After 175 Moonfalls, reposition the Beetom to chest height.",
+        "If needed, the Beetom can be left at the door while Samus moves to the farm bugs."
+      ]
+    },{
       "link": [1, 5],
-      "name": "Red Tower Spark (Come In Shinecharged)",
```

Comment:
Looks like "Red Tower Spark (Come In Shinecharged)" is getting accidentally deleted here?